pages/management/commands/convert_news_entries.py

```
@@ -11,12 +12,20 @@
-
 # Wagtail caps Page.slug at 255 characters; Entry.slug allows 300.
 WAGTAIL_SLUG_MAX_LENGTH = 255
 
 
+def _insert_line_breaks(string: str):
+    """
+    Insert two spaces before each return to guarantee that a linebreak is inserted.
+    This fixes examples such as `text`\r\n`text`, which are rendered as
+    two separate lines in V2, and a single line in V3.
+    """
+    reg = re.compile("[^\s]\\r\\n")
```

Ruff (0.16.3)

[warning] Invalid escape sequence: `\s`

Add backslash to escape sequence

(W605)